iml-wire-types/wire-types.rs

```diff
@@ -2280,6 +2268,7 @@ pub mod warp_drive {
         }
     }
 
+    #[allow(clippy::large_enum_variant)]
```

Comment:
I feel like we should probably fix the `large_enum_variant` warnings, but I'm not pushing a change because I know you're working in this area.